app/views/content_items/travel_advice.html.erb

```diff
@@ -16,6 +16,13 @@
   <div class="govuk-grid-column-two-thirds travel-advice__header">
     <%= render 'govuk_publishing_components/components/title', @content_item.title_and_context %>
 
+    <% @content_item.alert_status.each do |text| %>
+      <%= render "govuk_publishing_components/components/warning_text", 
+        text: text, 
+        country: @content_item.country_name 
```

Comment:
As the country name is being passed into the presenter (i.e. the `@content_item` object), I don't think that it also needs to be passed into the component here. `text` will already contain the country name. 

I also don't believe that the [warning text component](https://components.publishing.service.gov.uk/component-guide/warning_text) takes a "country" parameter.

Reply:
Thank you @leenagupte, I've now fixed this.